ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/JumpPass.scala

```diff
@@ -0,0 +1,30 @@
+package io.joern.ghidra2cpg.passes
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.EdgeTypes
+import io.shiftleft.passes.{DiffGraph, IntervalKeyPool, ParallelCpgPass}
+import io.shiftleft.semanticcpg.language._
+class JumpPass(cpg: Cpg, keyPool: IntervalKeyPool)
+    extends ParallelCpgPass[String](
+      cpg,
+      keyPools = Some(keyPool.split(1))
+    ) {
+  override def partIterator: Iterator[String] = List("").iterator
+
+  override def runOnPart(part: String): Iterator[DiffGraph] = {
+    implicit val diffGraph: DiffGraph.Builder = DiffGraph.newBuilder
+    cpg.call
+      .nameExact("<operator>.goto")
+      .where(_.argument.order(1).isLiteral)
+      .foreach { sourceCall =>
+        sourceCall.argument.order(1).code.l.headOption match {
+          case Some(destinationAddress) =>
+            cpg.call.lineNumber(Integer.parseInt(destinationAddress, 16)).foreach { destination =>
```

Comment:
Except if this is caught in the `lineNumber` method, I think this can still throw a `NumberFormatException` when attempting to parse the address (although now this _shouldn't_ happen anymore with the empty default being removed). Again there are different ways of dealing with this, but I just learnt that Scala strings have a `toIntOption` method which can be used to do the conversion safely :)

Reply:
Correction: `toLongOption` would be safer here, to account for 64 bit addresses. If these are just an offset, that's unlikely to make a difference, but I also don't see any reason not to do it.